en/flight_modes/flight_modes_summary.md

```diff
@@ -103,11 +125,8 @@ th {
  <td>M</td>
  <td></td>
  <td>
-<p>If zero roll/pitch sticks - vehicle levels out.</p>
-
-<p>If non-zero roll/pitch sticks - vehicle does a coordinated turn.</p>
-
-<p>Manual yaw input is added to rudder control input- to control sideslip.</p>
+  <p>RC/manual mode where centered RP sticks level vehicle (centered sticks put vehicle into straight and level flight). The vehicle course and altitude are not maintained, and can drift due to wind.</p>
+  <p>If roll/pitch sticks are non-zero the vehicle does a coordinated turn (manual yaw input is added to rudder control input to control sideslip). </p>
```

Comment:
I thought yaw mapped to the rudder? (ie it is the rudder that produces yaw?)

Reply:
This is correct as it's written. But you don't need a rudder to do a turn - if you have non-zero roll and adjust pitch accordingly you do a turn (thus changing yaw).